src/rtk/features/resourceApi.ts

```diff
@@ -34,19 +30,18 @@ export const resourceOwnerApi = createApi({
   }),
   tagTypes: ['APIs'],
   endpoints: (builder) => ({
-    getResourceOwners: builder.query<ResourceOwner[], resourceOwnerParams>({
-      query: (args) => {
-        const { resourceTypeList } = args;
+    getResourceOwners: builder.query<ResourceOwner[], ResourceType[] | void>({
+      query: (resourceTypeList) => {
         let resourceTypesUrl = '';
-        for (const type of resourceTypeList) {
+        resourceTypeList?.forEach((type) => {
           resourceTypesUrl = resourceTypesUrl + `&relevantResourceTypes=${type}`;
-        }
-        return `resources/getResourceOwners?${resourceTypesUrl}`;
+        });
+        return `resourceowners?${resourceTypesUrl}`;
```

Comment:
Not important, but could this be shortened down to something like this? 

```
return  resourceTypeList?.reduce((url, type) => {
  return url + `&relevantResourceTypes=${type}`;
}, '');
```

Reply:
That's a great suggestion! Thanks! 😄